egs/multi_en/s5/run.sh

```diff
@@ -137,19 +140,22 @@ if [ $stage -le 7 ]; then
   mfccdir=mfcc
   corpora="hub4_en fisher librispeech_100 librispeech_360 librispeech_500 swbd tedlium wsj"
   for c in $corpora; do
-    data=data/$c/train
-    steps/make_mfcc.sh --mfcc-config conf/mfcc.conf \
-      --cmd "$train_cmd" --nj 40 \
-      $data exp/make_mfcc/$c/train || exit 1;
-    steps/compute_cmvn_stats.sh \
-      $data exp/make_mfcc/$c/train || exit 1;
+    (
+     data=data/$c/train
+     steps/make_mfcc.sh --mfcc-config conf/mfcc.conf \
+       --cmd "$train_cmd" --nj 40 \
+       $data exp/make_mfcc/$c/train || touch $data/.error
+     steps/compute_cmvn_stats.sh \
+       $data exp/make_mfcc/$c/train || touch $data/.error
+    ) &
   done
+  wait
 fi
 
 # fix and validate training data directories
 if [ $stage -le 8 ]; then
   # get rid of spk2gender files because not all corpora have them
-  rm -f data/*/train/spk2gender
+  rm data/*/train/spk2gender 2>/dev/null
```

Comment:
There should be -f. Otherwise the script might exit because of set -e.

Reply:
I don't like using -f to make 'rm' be quiet, because it overrides when people change modes to save something.  Instead, do `|| true`